aptos-move/aptos-gas/src/transaction.rs

```diff
@@ -65,36 +65,36 @@ crate::params::define_gas_parameters!(
         [
             gas_unit_scaling_factor: GasScalingFactor,
             "gas_unit_scaling_factor",
-            1000
+            100
         ],
         // Gas Parameters for reading data from storage.
-        [load_data_base: InternalGas, "load_data.base", 1],
+        [load_data_base: InternalGas, "load_data.base", 16_000],
         [
             load_data_per_byte: InternalGasPerByte,
             "load_data.per_byte",
-            1
+            1_000
```

Comment:
@movekevin this is where we actually charge for loading resources from storage. This is assessed per resource accessed per transaction.

Reply:
How did you come up with 1000 gas units per byte?

Reply:
I borrowed the logic/ratio from EVM. In EVM, loading a word (32B) costs 100 gas units, while add costs 3. In our current implementation, the simpler instructions cost anywhere around 60 to 200, so I thought 1000 per byte might be reasonable.

I'm open to change it if you feel like's not right, and my hope is that we'll be able to fine tune it as we gather more data.

Reply:
Okay makes sense